backend/tools/import/appsettings.json

```diff
@@ -6,9 +6,13 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "Keycloak": {
+    "TokenUrl": "https://sso-dev.pathfinder.gov.bc.ca/auth/realms/quartech/protocol/openid-connect/token",
+    "ClientId": "pims-app"
+  },
   "Api": {
-    "Url": "http://pims-dev.pathfinder.gov.bc.ca/api/tools/import/properties",
-    "Method": "POST"
+    "ImportUrl2": "http://pims-dev.pathfinder.gov.bc.ca/api/tools/import/properties",
```

Comment:
/nit naming

Reply:
Ah forgot I made that change to test something.  Thanks for catching it.